services/lumberjack/bin/start-service.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+const Service = require('..');
+
+let service = new Service({
+  port: process.env.PORT,
+  influxHost: process.env.INFLUX_HOST,
+  influxPort: process.env.INFLUX_PORT,
+  pingHost: process.env.PING_HOST,
+  pingPort: process.env.PING_PORT,
+  forwardInteropHost: process.env.FORWARD_INTEROP_HOST,
+  forwardInteropPort: process.env.FORWARD_INTEROP_PORT,
+  telemetryHost: process.env.TELEMETRY_HOST,
+  telemetryPort: process.env.TELEMETRY_PORT,
```

Comment:
need to change this to `GROUND_TELEMETRY_HOST/PORT` and `PLANE_TELEMETRY_HOST/PORT`.

Reply:
host and port as separate env vars for clarification